### PR TITLE
Feature/vowel counting

### LINKED ISSUE #9: Feature 2: count number of vowels in text file
Counts number of vowels in provided text file and displays results

### LINKED ISSUE #13: Feature 2: vowel count test coverage
Add unit tests under tests package

### PR DESCRIPTION
**Pull request by Lightbrite88**
Created java class for counting vowels in a string as well as a class for junit tests.

### CODE REVIEW

File: src/fileAnalysis/AnalyzeFile.java [hunk 1]
```diff
@@ -8,9 +8,14 @@ public static void main(String [] args) {
     public static void analyzeFile() {
         try {
             String text = ReadFile.readFile("src/resources/SampleText.txt");
+<<<<<<< HEAD
```
Comment (rachellegelden) [inline src/fileAnalysis/AnalyzeFile.java:11]: Hmmm it looks like the merge conflict snuck in. Could you fix it again (just go into VS code and remove the <<<< HEAD, ====, and the >>>>>a9..., add the file AnalyzeFile.java, commit the change and push the branch again

File: src/fileAnalysis/CountVowels.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
+package fileAnalysis;
+
+public class CountVowels { //class for counting vowels
+    public static int countVowels(String inputString){ //static -> takes string, returns integer
+        if (inputString==null || inputString.isEmpty()){ //check for empty and null strings
+            return 0; //empty and null strings have no vowels
```
Comment (rachellegelden) [inline src/fileAnalysis/CountVowels.java:6]: 👍

File: src/fileAnalysis/CountVowels.java [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+            for (char ch : inputString.toCharArray()){ //look at each letter individually
+                if (isVowel(ch)){
+                    numVowels++; //if ch is a vowel, increment the number of vowels found
+                }
+            }
+            return numVowels;
+        } //close conditional
+    } //close static method
+
+    public static boolean isVowel(char ch){ //helper method for checking if character is a vowel
```
Comment (rachellegelden) [inline src/fileAnalysis/CountVowels.java:18]: 👍

**Review by rachellegelden — DISMISSED** [review]

File: src/fileAnalysis/AnalyzeFile.java (hunk 1 above)
Comment (rachellegelden) [inline src/fileAnalysis/AnalyzeFile.java]: Hmmm it looks like the merge conflict snuck in. Could you fix it again (just go into VS code and remove the <<<< HEAD, ====, and the >>>>>a9..., add the file AnalyzeFile.java, commit the change and push the branch again

**Review by rachellegelden — CHANGES_REQUESTED** [review]

File: src/fileAnalysis/AnalyzeFile.java (hunk 1 above)
Comment (Lightbrite88) [reply src/fileAnalysis/AnalyzeFile.java]: Did that work?

**Review by rachellegelden — APPROVED** [review]

**Pull request merged**